meta/autoid/autoid.go

```diff
@@ -511,13 +668,121 @@ func (alloc *allocator) alloc4Unsigned(tableID int64, n uint64, increment, offse
 	return min, alloc.base, nil
 }
 
+// alloc4Sequence is used to alloc value for sequence, there are several aspects different from autoid logic.
+// 1: sequence allocation don't need check rebase.
+// 2: sequence allocation don't need auto step.
+// 3: sequence allocation may have negative growth.
+// 4: sequence allocation batch length can be dissatisfied.
+// 5: sequence batch allocation will be consumed immediately.
+func (alloc *allocator) alloc4Sequence(tableID int64) (int64, int64, int64, error) {
```

Comment:
Better for readability
```suggestion
func (alloc *allocator) alloc4Sequence(tableID int64) (min int64, max int64, round int64, err error) {
```

Reply:
nice idea!